TrackletAlgorithm/MatchCalculator.h

```diff
@@ -752,7 +752,8 @@ void MatchCalculator(BXType bx,
     ap_int<12> z_corr        = (full_z_corr + (1<<(kZ_corr_shift-1))) >> kZ_corr_shift; // only keep needed bits
      
     // Apply the corrections
-    ap_int<15> proj_phi_corr = proj_phi + phi_corr;  // original proj phi plus phi correction
+    const int kProj_phi_len = (APTYPE == BARRELPS) ? 15 : 18;
```

Comment:
@bryates Would it make sense to replace the right-hand side with something like `AllProjection<APTYPE>::kAProjPhiSize + 1`, so as to avoid magic numbers?

Reply:
Good suggestion, I just pushed the change.
https://github.com/cms-L1TK/firmware-hls/blob/0f406a2e1e671cf9a23891154579b9d79060a591/TrackletAlgorithm/MatchCalculator.h#L756

Reply:
Thanks @bryates.